examples/jsm/renderers/webgpu/nodes/WebGPUNodes.js

```diff
@@ -0,0 +1,38 @@
+
+class WebGPUNodes {
+
+	constructor( ) {
+
+		this.uniformsData = new WeakMap();
+
+	}
+
+	get( object ) {
+
+		let data = this.uniformsData.get( object );
+
+		if ( data === undefined ) {
+			
+			this.uniformsData.set( object, data );
```

Comment:
I guess you the following line is missing, otherwise `undefined` is set:
```js
data = {};
```
It seems `WebGPUNodes` is not yet used in the renderer. What are your planes for this class?

Reply:
Compute `bindings` replacing functions like `WebGPUBindings._getMeshBasicBindings()`, get all `nodes` of a material for update in each `frame` and soon generate `hash` per material...